scripts/ui/topological-ui-dev.sh

```diff
@@ -0,0 +1,32 @@
+BASE_DIR=~/Projects
+mkdir -p $BASE_DIR
+
+cd $BASE_DIR
+if [ ! -d insights-proxy ]; then
+  git clone git@github.com:RedHatInsights/insights-proxy.git
+  cd insights-proxy
+  npm install
```

Comment:
@martinpovolny is there some specific version of npm required?

Reply:
I do not know what is the minimal version, but I am not aware of a specific version requirement.